en/docs/setup/migrating-userstore-managers.md

```diff
@@ -0,0 +1,72 @@
+# Migrating User Stores
+
+Before doing the actual user store migration, it is recommended to do a dry run and analyze the generated report for 
+any recommendations related to the migration. 
+
+## Dry Run
+
+Dry-run capability in the user store migrator will allow the migration utility to validate the system for the 
+user store configurations and generate a report regarding the migration compatibility. In the generated migration report, 
+if there are any warnings it is recommended to contact the WSO2 support to identify the best migration strategy.
+
+### How to Run
+
+Configure the migration report path using the value of “reportPath” in the migration-config.yaml. 
+This path should be an absolute path. 
+
+Run the migration utility with system property “dryRun”. 
+
+Ex:
+
+Unix/Linux
+
+``` bash
+sh wso2server.sh -Dmigrate -Dcomponent=identity -DdryRun
+```
+
+Windows
+
+``` bash
+wso2server.bat -Dmigrate -Dcomponent=identity -DdryRun
+```
+ 
+After that analyze the generated report that resides in the provided location.
+
+## Migrating User Stores
+
+If you have multiple tenants and multiple user stores and you just only need to migrate a few of them, refer to the 
+configuration section below on how to migrate few of them. If you need to migrate all of them, use the “migrateAll” 
```

Comment:
Remove "just" in 
...and you only need to migrate a few of the...

Remove "on how to migrate few of them"

use the `migrateAll`